app/assets/scripts/components/common/layers/types.js

```diff
@@ -214,6 +214,16 @@ export const layerTypes = {
       const vectorData = vector.data.replace('{date}', formatDate);
       const rasterTiles = raster.tiles.map(tile => tile.replace('{date}', formatDate));
 
+      // inference data moves around, recenter on each update
+      fetch(vectorData)
+        .then(res => res.json())
+        .then(geo => {
+          mbMap.fitBounds(bbox(geo));
+        })
+        .catch(err => {
+          throw err;
```

Comment:
Throwing this error here, will result in an `Uncaught Promise Error`, and nothing will happen.
Would be better to just log it.